src/Microsoft.TemplateEngine.Abstractions/PublicAPI.Shipped.txt

```diff
@@ -117,9 +116,6 @@ Microsoft.TemplateEngine.Abstractions.IPostAction.Description.get -> string?
 Microsoft.TemplateEngine.Abstractions.IPostAction.ManualInstructions.get -> string?
 Microsoft.TemplateEngine.Abstractions.IPostActionLocalizationModel
 Microsoft.TemplateEngine.Abstractions.IPostActionLocalizationModel.Description.get -> string?
-Microsoft.TemplateEngine.Abstractions.IPostActionLocalizationModel.Instructions.get -> System.Collections.Generic.IReadOnlyDictionary<int, string!>!
-Microsoft.TemplateEngine.Abstractions.IPrioritizedComponent
```

Comment:
Would there be a CI error if this file was wrong?
I just noticed the error during build and used "Auto Fix for solution".

Reply:
I think it might be a bug in analyzer then. 
It removed this line from `shipped` and put it to `unshipped`, though there is no change to the actual file.

Reply:
I think I understand the problem. I accidentally removed it while removing the others. Then "Auto Fix" automatically added into the "Unshipped" file.
Could it be that we should first put API changes to Unshipped file and at the end of a release, we move them all into the "Shipped" file? I'm not sure if that is the reason why we have both shipped and unshipped files, but this could be used to easily what changes are being made to the API in that particular release.
Today, all of our unshipped files are empty and every change is put into shipped files.

Reply:
That's correct workflow:
- changing existing API (breaking change) will modify shipped file
- introducing new API (not a breaking change) will add it to unshipped
Once we ship the release, we copy over unshipped -> shipped, and changing it since that moment is a breaking change.